vowpalwabbit/parser.cc

```diff
@@ -590,8 +595,7 @@ void enable_sources(VW::workspace& all, bool quiet, size_t passes, input_options
       catch (std::exception const&)
       {
         // when trying to fix this exception, consider that an empty filename_to_read is valid if all.stdin_off is false
-        if (!filename_to_read.empty())
-        { *(all.trace_message) << "can't open '" << filename_to_read << "', sailing on!" << endl; }
+        if (!filename_to_read.empty()) { all.logger.err_error("can't open '{}', sailing on", filename_to_read); }
```

Comment:
Would it be too radical to change this?

Reply:
Also something to think about is if people are currently looking for `sailing on!` with the exclamation point in their code like here:
https://github.com/VowpalWabbit/vowpal_wabbit/issues/1895

Reply:
Alright I'll leave the `!` in there